Makefile

```diff
@@ -46,12 +46,21 @@ publish-stage: ## Only use in an emergency
 run-dev:  ## Runs the task in dev - see readme for more info
 	aws ecs run-task --cluster DSS-SubmissionService-dev --task-definition DSS-SubmissionService-dev-task --network-configuration "awsvpcConfiguration={subnets=[subnet-0488e4996ddc8365b,subnet-022e9ea19f5f93e65],securityGroups=[sg-044033bf5f102c544],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1
 
+verify-dspace-connnection-dev: # Verify dev app can connect to DSpace
```

Comment:
I think small typo here, triple "n", maybe should be: `verify-dspace-connection-dev`